### PR TITLE
TST: ignore sentinel deprecation warning from traitlets

### PR DESCRIPTION
**Pull request by kecnry**
### Description

This pull request ignores the deprecation warning from traitlets:

```
E   DeprecationWarning:
E               Sentinel is not a public part of the traitlets API.
E               It was published by mistake, and may be removed in the future.
```

looks like bqplot imports traittypes, which has an `import *`, which includes the line `Empty = Sentinel('Empty', 'traittypes',` which is triggering the deprecation warning 🤔 

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Comment by pllim** [comment]
That's not good. `bqplot` imports `traittypes` that causes this warning.

https://github.com/jupyter-widgets/traittypes/blob/af2ebeec9e58b73a12d4cf841bd506d6eadb8868/traittypes/traittypes.py#L21

That repo has no activities the last 3 years.

**Comment by kecnry** [comment]
Right, although I suspect this was caused by a recent release of traitlets.  It looks like they just raise their own deprecation warning on import?

**Comment by pllim** [comment]
> It looks like they just raise their own deprecation warning on import?

No, the warning is from traittypes, not trailets.

* https://github.com/jupyter-widgets/traittypes/issues/47
* https://github.com/jupyter-widgets/traittypes/issues/48

File: pyproject.toml [hunk 1]
```diff
@@ -136,6 +136,7 @@ filterwarnings = [
     "ignore:zmq\\.eventloop\\.ioloop is deprecated in pyzmq:DeprecationWarning",
     "ignore:The unit 'Angstrom' has been deprecated in the VOUnit standard\\. Suggested.* 0\\.1nm\\.",
     "ignore:specutils uses the deprecated entry point asdf_extensions",
+    "ignore:.*Sentinel is not a public part of the traitlets API.*",
```
Comment (pllim) [inline pyproject.toml:139]: Need to find regex incantation that includes endline at the beginning.

**Review by pllim — APPROVED** [review]
The warning is worrisome but nothing we can do here. I reported it upstream. Thanks!

**Pull request merged**